src/components/components/ebay-notice-base/style.js

```diff
@@ -0,0 +1 @@
+require('@ebay/skin/icon');
```

Comment:
This shouldn't be necessary since we are using the `<ebay-icon>` and it will bring it in.